server/services/openweather/lib/formatResults.js

```diff
@@ -1,3 +1,36 @@
+/**
+ * @description Transform OpenWeather weather information to gladys weather information.
+ * @param {string} weather - OpenWeather weather information.
+ * @returns {Object} Return gladys weather information.
+ * @example
+ * const weather = translateIconToWeather(weather);
+ */
+
+const translateWeatherOWToGladys = (weatherInformation) => {
+  if (weatherInformation.main.search('Snow') !== -1) {
```

Comment:
Can the `weatherInformation.main` contains multiple values?

Maybe, instead of calling `search`, we call use `startsWith` to improve performances.

For more, I think we'll get more readibility with a switch case instead of big if list.

Reply:
It was like that in current implementation. I think the main is a text (we don't have that much information on what are the possible values if I remember right)

But if OpenWeather API specify all values in another fields I agree it's better with a switch case (to check on OpenWeather documentation)

Reply:
I have check the OpenWeather API, and I have change the code to accord to the documentation : https://openweathermap.org/weather-conditions#Weather-Condition-Codes-2